source/adminguide/templates.rst

```diff
@@ -275,13 +275,19 @@ To upload a template:
 
 .. include:: templates/_bypass-secondary-storage-kvm.rst
 
-Uploading Templates from a local computer
+Uploading Templates and ISOs from a local computer
 -------------------------------------------
 
-It's also possible to upload an already prepared template from your local computer.
-The steps are similar as when Uploading a template from a remote HTTP server, except that you need to choose a local template file from your PC.
+It's also possible to upload an already prepared template or an ISO from your local computer.
+The steps are similar as when Uploading a template/ISO from a remote HTTP server, except that you need to choose a local template/ISO file from your PC.
 For this feature to work, your SSVMs must be supporting HTTPS (for more info please visit `“Using a SSL Certificate for the Console Proxy” 
-<systemvm.html#using-a-ssl-certificate-for-the-console-proxy>`_.)
+<systemvm.html#using-a-ssl-certificate-for-the-console-proxy>`_).
+
+Example GUI dialog of uploading (ISO) from local (browser) is given below:
```

Comment:
```suggestion
Example UI dialog of uploading (ISO) from local (browser) is given below:
```
Do we always mention our GUI as UI? And maybe we don't even have mention this line as other parts of doc simply adds screenshots without explicitly mentioning them.